Makefile

```diff
@@ -30,6 +31,16 @@ ifeq ($(enable_fips), true)
 	AMI_VARIANT := $(AMI_VARIANT)-fips
 endif
 
+ifeq ($(os_distro), al2023)
+	ifdef accelerator_vendor
+		AMI_VARIANT := $(AMI_VARIANT)-$(accelerator_vendor)
+		ifeq ($(accelerator_vendor), nvidia)
+			nvidia_major_driver_version := $(NVIDIA_MAJOR_DRIVER_VERSION_DEFAULT)
```

Comment:
The user can't override this without `?=`. I'm not sure we want this in the top level Makefile anyway -- if the user really wants the driver version to appear in the AMI name, they can set the `ami_name` variable to whatever they want. The default value for the driver version var should be defined in the `variables-default.json`.

Reply:
> The user can't override this without `?=`.

Not sure what you mean by that, one can override the `nvidia_major_driver_version` like so:

```
make k8s=1.28 os_distro=al2023 accelerator_vendor=nvidia nvidia_major_driver_version=560
```

But since you're the second reviewer to point this out and I totally missed the fact that `ami_name` can also be set, I'll put the version in `variables-default.json`.